src/components/MoneyRequestConfirmationList.js

```diff
@@ -273,6 +278,36 @@ function MoneyRequestConfirmationList(props) {
         );
     }, [confirm, props.selectedParticipants, props.bankAccountRoute, props.iouCurrencyCode, props.iouType, props.isReadOnly, props.policyID, selectedParticipants, splitOrRequestOptions]);
 
+    /**
+     * Grab the appropriate image URI based on file type
+     *
+     * @param {String} receiptPath
+     * @param {String} receiptSource
+     * @returns {*}
+     */
+    const getImageURI = (receiptPath, receiptSource) => {
+        const {fileExtension} = FileUtils.splitExtensionFromFileName(receiptSource);
+        const isReceiptImage = Str.isImage(props.receiptSource);
+
+        if (isReceiptImage) {
+            return receiptPath;
+        }
+
+        if (fileExtension === 'html') {
+            return ReceiptHTML;
```

Comment:
NAB but a good idea for a follow-up, we should have our server go fetch metadata for HTML receipts and use that to display a link preview instead of a generic `ReceiptHTML` image.

Reply:
An issue was created for this: https://github.com/Expensify/Expensify/issues/304304